### ISSUE TITLE
Use pad-start and pad-end

### ISSUE THREAD

**Issue by satazor** [gillesdemey/cli-progress-bar#5]
Instead of the deprecated padleft and padright from lodash:

```
npm WARN deprecated lodash.padright@3.1.3: This package has been discontinued in favor of lodash.padend@^4.0.0.
npm WARN deprecated lodash.padleft@3.1.3: This package has been discontinued in favor of lodash.padstart@^4.0.0.
```

**Comment by gillesdemey** [comment]
I accept pull requests 😊

**Comment by satazor** [comment]
Done, see #6

**Comment by satazor** [comment]
Bump

**Comment by gillesdemey** [comment]
Fixed with #6, publishes as `v1.0.1`
